### DISCUSSION TITLE
Locais de trabalho para Dev Nômade

### DISCUSSION THREAD

**Discussion by luisfmsouza** [frontendbr/forum#241] [Discussão]
O @diegoemerique puxou o assunto sobre equipamentos pra Dev Nômade na issue #237.

E falando de Dev Nômade/trabalho remoto, outro assunto recorrente é de bons lugares pra trabalhar quando não estamos na nossa cidade, ou até mesmo quando não conhecemos bem a nossa região.

Um bom lugar pra trabalhar precisa de boa internet e preço justo(ou ser grátis).

**Conhecem algum lugar bacana pra trabalhar? Ou até mesmo um lugar ruim que devemos evitar?**

Ex:

**Starbucks**
https://goo.gl/maps/ddZhVudJUUR2.
Cidade: Campinas 
Preço: Grátis
Qualidade da internet: razoável

**Comment by felipefialho** [comment]
O @willianjusten, @diogomoretti, @afonsopacifer, @ZeuCxb e eu ficamos em um Starbucks na Jesuíno Arruda para trabalharmos juntos durante a Front End Week.

**Starbucks**
Cidade: São Paulo - SP
[Endereço](https://www.google.com.br/maps/place/Starbucks/@-23.5794765,-46.6771247,15z/data=!4m8!1m2!2m1!1sstarbucks!3m4!1s0x0:0xb6ae6842d9ea7ede!8m2!3d-23.5823266!4d-46.6766998)
Preço: Grátis
Qualidade da internet: Boa

A vantagem é que tem estacionamento. 

Gostei bastante.

**Comment by Raffesmind** [comment]
Um lugar que pode ser bem bacana se planeja ficar o dia todo ou chegar cedo nos lugares é o **Campus** do Google:

**Campus**
Cidade: São Paulo - SP
Endereço: Rua Coronel Oscar Porto, 70 - Paraíso, São Paulo - SP, 01313-902 (Duas quadras do metro brigadeiro, bem próximo ao metro)
Preço: Grátis
Qualidade da Internet: Boa

Obs: É necessário preencher um pequeno cadastro pra ter acesso ao campus: [Link](https://www.campus.co/sao-paulo/pt/sign-upl)

**Comment by vicainelli** [comment]
### Coworking Spaces FIAP SU

[fiap.com.br/coworking](https://www.fiap.com.br/coworking/)

#### Av. Paulista

**Endereço:** Av. Paulista, 1.106 – 7° andar
**Horário:** Segunda a sexta, das 9 às 18 horas.
**Preço:** Grátis

#### Vl. Olímpia

**Endereço:** Rua Olimpíadas, 186 – Vila Olímpia
**Horário:** Segunda a sexta, das 9 às 18 horas.
**Preço:** Grátis

> **IMPORTANTE** – Para utilizar o Coworking você precisa de autorização prévia. Acesse a página de [agendamento](https://www.fiap.com.br/coworking/#agendamento) para mais informações.

**Comment by felipefialho** [comment]
https://github.com/frontendbr/open-source/issues/12

**Comment by herberthenrique** [comment]
Hoje eu encontrei esse app, que parece sensacional.
Chama **workfrom**, vocês podem achar lugares pelo [site](https://workfrom.co/)
ou pelos [apps](https://workfrom.co/download).

**Comment by bukinoshita** [comment]
Sendo nomade quase 2 anos já, o lugar que sempre procuro é o Starbucks. Apesar do barulho e sempre estar lotado, Starbucks tem uma internet boa, lugar confortável e agradável.

Além do Starbucks, alguns aeroportos (caso esteja em um) também tem uma internet bem razoável, mas em cafeterias, restaurantes, e não o wifi publico.

Em alguns países é bem comum ter Wifi em Shopping, me surpreendi uma vez que estava em Punta Del Este e encontrei um shopping no meio do nada com wifi a 40mbps.

Eu geralmente vou em Starbucks e quando estou afim de explorar mais eu procuro outras cafeterias no Foursquare. Aproveitar pra trabalhar e conhecer.
- [Nomadlist CLI](https://github.com/BuKinoshita/nomadlist-cli) - Fiz uma versão CLI do Nomadlist, quem é Dev pode curtir hehe
- [Nomadlist](https://nomadlist.com) - Ver a velocidade da internet em cada cidade
- [Places to work](https://placestowork.co/) - Do mesmo criado do nomadlist
- [Cafe 4 Nomads](http://cafes4nomads.com) - Lista de cafés com internet
- [Workfrom](https://workfrom.co) - Citado acima

Se nada disso satisfazer, o melhor é procurar Co-working spaces.
